config/routes.rb

```diff
@@ -1,30 +1,20 @@
 Rails.application.routes.draw do
   get "orders/create"
-
-  # Example resource route
-  # (maps HTTP verbs to controller actions automatically):
-
   resources :products
 
-  # The priority is based upon order of creation:
-  # first created -> highest priority.
-  # See how all your routes lay out with "rake routes".
+  root "landing#index"
 
-  # You can have the root of your site routed with "root"
-  # root 'welcome#index'
-  root to: "landing#index"
+  get "/review"             => "products#review"
+  post "/review"            => "products#rate"
 
-  get "/review", to: "products#review"
-  post "/review", to: "products#rate"
-
-  get "/checkout", to: "checkout#index"
-  get "/search", to: "search#result"
-  get "/filter", to: "filter#index"
-  get "/cart", to: "carts#index", as: :cart_index
-  post "/cart", to: "carts#add_item"
-  get "/cart/delete-all", to: "carts#delete_all", as: :empty_cart
-  get "/cart/delete/:id", to: "carts#delete_item", as: :delete_order_item
-  post "/cart/update/:id", to: "carts#update_item", as: :update_order_item
+  get "/checkout"           => "checkout#index"
+  get "/search"             => "search#result"
+  get "/filter"             => "filter#index"
+  get "/cart"               => "carts#index", as: :cart_index
+  post "/cart"              => "carts#add_item"
+  get "/cart/delete-all"    => "carts#delete_all", as: :empty_cart
```

Comment:
Why do we have an endpoint with `delete-all`?
We could have `get "/cart/delete"` instead.